guide/raids/amirdrassil.md

```diff
@@ -0,0 +1,445 @@
+</iframe>
+</div>
+
+#### Fire T31:
+<div class="iframe-holder">
+<iframe src="https://www.raidbots.com/simbot/render/talents/BYQAAAAAAAAAAAAAAAAAAAAAAAAAAAAg0SrUSCgDIlk0oElUSCRAAAAAAKBkUSCaCItkkGCIJJEI?width=530&level=70" frameborder="0" width="530px" height="100%">
+</iframe>
+</div>
+
+<hr>
+
+## Fight Breakdown
+
+#### Damage:
+* Focus down {{ site.data.raid.amirdrassil.cycle_warden }} and {{ site.data.raid.amirdrassil.manifested_dream }} (Mythic-only) whenever they are active.
+* Use {{ site.data.talent.swg }} and {{ site.data.talent.gow }} to maintain uptime whilst being targeted by {{ site.data.raid.amirdrassil.threads_of_life }} or dodging {{ site.data.raid.amirdrassil.lucid_exhaust }} (Mythic-only).
+
+#### Defensives:
+* Use {{ site.data.talent.as }} or healing consumables for overlaps of {{ site.data.raid.amirdrassil.continuum }} and {{ site.data.raid.amirdrassil.lucid_miasma }} (Mythic-only) or when accumulating high stacks of {{ site.data.raid.amirdrassil.verdant_matrix }}.
+* Use {{ site.data.talent.ag }} during {{ site.data.raid.amirdrassil.continuum }}.
+* If talented, use {{ site.data.talent.earth_shield }} and {{ site.data.talent.healing_stream_totem }} as a movement filler throughout the fight.
+* Use {{ site.data.spell.healing_surge }} if necessary.
+
+#### Utilities:
+* Use {{ site.data.talent.wrt }} to help players repositioning to kill {{ site.data.raid.amirdrassil.cycle_warden }} or {{ site.data.raid.amirdrassil.manifested_dream }} (Mythic-only) as fast as possible.
+* Use {{ site.data.spell.wind_shear }} to interrupt {{ site.data.raid.amirdrassil.nature_volley }} from {{ site.data.raid.amirdrassil.cycle_warden }}.
+
+</div>
+</div>
+</div>
+
+<div class="card">
+<div class="card-header" id="smolderon">
+<div data-toggle="collapse" data-target="#smolderon-collapse" aria-expanded="true" aria-controls="smolderon-collapse" class="raid-header amirdrassil_the_dreams_hope7"><h2>Smolderon</h2></div>
+</div>
+<div id="smolderon-collapse" class="collapse" aria-labelledby="smolderon" data-parent="#accordion">
+<div class="card-body" markdown="1">
+
+## Boss Fight Profile: Single Target
+
+#### Talents:
+
+#### Lightning T30:
+<div class="iframe-holder">
+<iframe src="https://www.raidbots.com/simbot/render/talents/BYQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAoVSSLJUSSLRDRJQiEAAAAAgSAJlkgmASLJpdACIJBBC?width=530&level=70" frameborder="0" width="530px" height="100%">
+</iframe>
+</div>
+
+#### Fire T31:
+<div class="iframe-holder">
+<iframe src="https://www.raidbots.com/simbot/render/talents/BYQAAAAAAAAAAAAAAAAAAAAAAAAAAAAg0SrUSCgDIlk0oElUSCRAAAAAAKBkUSCaASLJpdACIJJEI?width=530&level=70" frameborder="0" width="530px" height="100%">
+</iframe>
+</div>
+
+<hr>
+
+## Fight Breakdown
+
+#### Damage:
+* Ensure to maximize your output while benefiting from {{ site.data.raid.amirdrassil.ignited_essence }}.
```

Comment:
Similar to Gnarlroot, I would specifically say to hold PW and SK for that phase ?